### ISSUE TITLE
 Train the model

### ISSUE THREAD

**Issue by SeppeWillems13** [howest-gp-ipr/st-2223-1-d-ee-SeppeWillems13#6]
## Train a hand gesture recognition model

- [x] Split the dataset into training and test sets
- [x] Choose a machine learning model (e.g. convolutional neural network)
- [x] Train the model using TensorFlow or another machine-learning library
- [x] Test the model using the test set to evaluate its performance

**Comment by SeppeWillems13** [comment]
Temp model trained: could be improved in the future with larger dataset
